iquip/apps/builder.py

```diff
@@ -297,8 +297,13 @@ class _ScanEntry(QWidget):
     
     Attributes:
         name: The name of the scannable object.
-        state: The dictionary that describes argument information of each scan type.
-        stackWidget: The QstackWidget that contains widget of each scan type.
+        state: Each key and its value are:
```

Comment:
I think there are too many columns, hence how about "Each key and its value are as follows."?